packages/block-library/src/template-part/edit/selection/template-part-previews.js

```diff
@@ -203,20 +197,12 @@ export default function TemplatePartPreviews( {
 } ) {
 	const composite = useCompositeState();
 	const templateParts = useSelect( ( select ) => {
-		const publishedTemplateParts =
-			select( 'core' ).getEntityRecords( 'postType', 'wp_template_part', {
-				status: [ 'publish' ],
-				per_page: -1,
-			} ) || [];
-
-		const currentTheme = select( 'core' ).getCurrentTheme()?.stylesheet;
-		const themeTemplateParts =
-			select( 'core' ).getEntityRecords( 'postType', 'wp_template_part', {
-				theme: currentTheme,
-				status: [ 'auto-draft' ],
-				per_page: -1,
-			} ) || [];
-		return [ ...themeTemplateParts, ...publishedTemplateParts ];
+		return (
+			select( 'core' ).getEntityRecords(
+				'postType',
+				'wp_template_part'
+			) || []
+		);
```

Comment:
Shouldn't we still limit this to the current theme's template parts?

Reply:
they're just part of the fetch all behavior.